hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -1849,7 +1849,7 @@ public ExpiredOpenKeys getExpiredOpenKeys(Duration expireThreshold,
             final KeyArgs.Builder keyArgs = KeyArgs.newBuilder()
                 .setVolumeName(info.getVolumeName())
                 .setBucketName(info.getBucketName())
-                .setKeyName(info.getKeyName())
+                .setKeyName(openKeyInfo.getKeyName())
```

Comment:
Please a comment here, explain why use openKeyInfo.getKeyName() instead of info.getKeyName.